jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2220,7 +2235,13 @@ void java_bytecode_convert_methodt::convert_invoke(
     symbol_table.add(symbol);
   }
 
-  if(is_virtual)
+  if(is_enum_values_clone_call)
+  {
+    const std::string clone_name =
+      "java::array[" + calling_class_name + "].clone:()Ljava/lang/Object;";
```

Comment:
I don't think you can add to a string literal. You should use `std::string("java::array[") + ...` instead